### PR TITLE
1253 add help section

### PR DESCRIPTION
**Pull request by ramtob**

### CODE REVIEW

File: src/app/@ansyn/menu/models/menu-session-state.model.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export interface MenuSessionState {
 	selectedMenuItem?: string;
 	isPinned?: boolean;
+	doneStartupOperations?: boolean
```
Comment (yairtawil) [inline src/app/@ansyn/menu/models/menu-session-state.model.ts]: remove

File: src/app/@ansyn/menu/reducers/menu.reducer.ts [hunk 2]
```diff
@@ -10,19 +10,29 @@ import { createFeatureSelector, createSelector, MemoizedSelector } from '@ngrx/s
...
 	selectedMenuItem: string;
 	isPinned: boolean;
 	autoClose: boolean;
+	showHelpOnStartup: boolean;
+	doneStartupOperations: boolean;
 }
 
 export const initialMenuState: IMenuState = menuItemsAdapter.getInitialState({
-	selectedMenuItem: sessionData().selectedMenuItem,
+	selectedMenuItem: (sessionData().doneStartupOperations ?
```
Comment (yairtawil) [inline src/app/@ansyn/menu/reducers/menu.reducer.ts]:
```typescript
isPinned : sessionData() && sessionData().isPinned 
selectedMenuItem: sessionData() && sessionData().selectedMenuItem
```

File: src/app/@ansyn/menu/reducers/menu.reducer.ts [hunk 3]
```diff
@@ -61,6 +71,15 @@ export function MenuReducer(state: IMenuState = initialMenuState, action: MenuAc
 
 		case MenuActionTypes.SET_AUTO_CLODE:
 			return { ...state, autoClose: action.payload };
+
```
Comment (yairtawil) [inline src/app/@ansyn/menu/reducers/menu.reducer.ts]: remove from menu

File: src/app/@ansyn/ansyn/ansyn.menu-items.ts [hunk 4]
```diff
@@ -42,5 +43,11 @@ export const ansynMenuItems: MenuItem[] = [
 		component: SettingsComponent,
 		iconClass: 'icon-main-settings',
 		production: false
+	},
+	{
+		name: 'Help',
+		component: HelpComponent,
+		iconClass: 'icon-help-settings',
+		production: false
```
[the remark this answers is not in the document]
Comment (ramtob) [reply src/app/@ansyn/ansyn/ansyn.menu-items.ts]: done

File: src/app/@ansyn/menu-items/help/components/help.component.html [hunk 5]
```diff
@@ -0,0 +1,98 @@
+<carousel [interval]="0">
+	<slide>
```
[the remark this answers is not in the document]
Comment (ramtob) [reply src/app/@ansyn/menu-items/help/components/help.component.html]: done

File: src/app/@ansyn/menu-items/help/components/help.component.less [hunk 6]
```diff
@@ -0,0 +1,77 @@
+@import "../../../core/less/common";
```
[the remark this answers is not in the document]
Comment (ramtob) [reply src/app/@ansyn/menu-items/help/components/help.component.less]: done

File: src/app/@ansyn/menu-items/help/components/help.component.ts [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+import { SetShowHelpOnStartup } from '@ansyn/menu/actions/menu.actions';
+
+@Component({
+	selector: 'ansyn-help',
+	templateUrl: './help.component.html',
+	styleUrls: ['./help.component.less']
+})
+export class HelpComponent {
+
+	public showHelpOnStartup$ = this.store.select<IMenuState>(menuStateSelector)
```
[the remark this answers is not in the document]
Comment (ramtob) [reply src/app/@ansyn/menu-items/help/components/help.component.ts]: done

File: src/app/@ansyn/menu-items/help/index.ts [hunk 8]
```diff
@@ -0,0 +1 @@
+export { HelpModule } from './help.module';
```
[the remark this answers is not in the document]
Comment (ramtob) [reply src/app/@ansyn/menu-items/help/index.ts]: done

File: src/app/@ansyn/menu-items/help/help.module.ts [hunk 9]
```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { HelpComponent } from './components/help.component';
+import { CarouselModule } from 'ngx-bootstrap/carousel';
+import { CommonModule } from '@angular/common';
+import { CoreModule } from '@ansyn/core/core.module';
+
+@NgModule({
```
[the remark this answers is not in the document]
Comment (ramtob) [reply src/app/@ansyn/menu-items/help/help.module.ts]: done

File: src/app/@ansyn/menu/actions/menu.actions.ts [hunk 10]
```diff
@@ -11,7 +11,9 @@ export const MenuActionTypes = {
 	TRIGGER: {
 		CONTAINER_CHANGED: 'CONTAINER_CHANGED'
 	},
-	SET_AUTO_CLODE: 'SET_AUTO_CLODE'
+	SET_AUTO_CLODE: 'SET_AUTO_CLODE',
+	SET_SHOW_HELP_ON_STARTUP: 'SET_SHOW_HELP_ON_STARTUP',
```
[the remark this answers is not in the document]
Comment (ramtob) [reply src/app/@ansyn/menu/actions/menu.actions.ts]: done

File: src/app/@ansyn/menu/menu/menu.component.less [hunk 11]
```diff
@@ -1,6 +1,8 @@
-@import "core/less/common";
-@import 'core/less/z-index-globals';
-@import 'core/less/tooltip';
+@import "../../core/less/common";
```
[the remark this answers is not in the document]
Comment (ramtob) [reply src/app/@ansyn/menu/menu/menu.component.less]: done

File: src/app/@ansyn/menu/helpers/menu-local-storage.helper.ts [hunk 12]
```diff
@@ -0,0 +1,12 @@
+import { MenuLocalStorageState } from '@ansyn/menu/models/menu-local-storage-state.model';
```
[the remark this answers is not in the document]
Comment (ramtob) [reply src/app/@ansyn/menu/helpers/menu-local-storage.helper.ts]: done

File: src/app/@ansyn/menu/models/menu-session-state.model.ts (hunk 1 above)
Comment (ramtob) [reply src/app/@ansyn/menu/models/menu-session-state.model.ts]: done

File: src/app/@ansyn/menu/reducers/menu.reducer.ts (hunk 2 above)
Comment (ramtob) [reply src/app/@ansyn/menu/reducers/menu.reducer.ts]: done

File: src/app/@ansyn/menu/reducers/menu.reducer.ts (hunk 3 above)
Comment (ramtob) [reply src/app/@ansyn/menu/reducers/menu.reducer.ts]: done

**Pull request merged**